src/components/Bars/components/BarLoading.tsx

```diff
@@ -0,0 +1,35 @@
+import { cx } from "@/utils/cx/cx";
+import { useGraph } from "@/hooks/use-graph/use-graph";
+import { useId } from "react";
+
+export const BarLoading = () => {
+	const maskId = useId();
+	const { viewbox } = useGraph();
+	const CX = viewbox.x / 2;
+	const CY = viewbox.y / 2;
+
+	const BAR_GAP = 20;
+	const BAR_WIDTH = viewbox.x * 0.08;
+	return (
+		<svg
+			id="loading"
+			role="status"
+			aria-busy={true}
+			viewBox={`0 0 ${viewbox.x} ${viewbox.y}`}
+			className={cx("[grid-area:graph] h-full w-full")}
+			preserveAspectRatio={"none"}
+		>
+			<path
+				vectorEffect={"non-scaling-stroke"}
+				d={`
+					M${CX - BAR_WIDTH - BAR_GAP - BAR_WIDTH / 2} ${viewbox.y} L${CX - BAR_WIDTH - BAR_GAP - BAR_WIDTH / 2} 2000 L${CX - BAR_WIDTH - BAR_GAP + BAR_WIDTH - BAR_WIDTH / 2} 2000 L${CX - BAR_WIDTH - BAR_GAP + BAR_WIDTH - BAR_WIDTH / 2} 3000
```

Comment:
Also calculate these last hardcoded numbers.

I.E can see a few 1600 3000 1000 numbers in this string.

`1000 = viewbox.<axis> * 0.333`
`3000` = viewbox.<x>
`1600` = .... etc